app/src/main/java/com/example/securityapplication/InputValidation.java

```diff
@@ -0,0 +1,26 @@
+package com.example.securityapplication;
+
+
+import android.content.Context;
+import android.support.design.widget.TextInputEditText;
+import android.support.design.widget.TextInputLayout;
+
+public class InputValidation {
+    private Context context;
+    public InputValidation(Context context) {
+        this.context = context;
+    }
+
+    public boolean is_Empty(TextInputEditText textInputEditText,
+            TextInputLayout textInputLayout,String message){
+        String value = textInputEditText.getText().toString().trim();
+        if(value.isEmpty()){
+            textInputLayout.setError(message);
```

Comment:
Add appropriate error for each field like phone number , aadhar , location rather just using generic message as it is not very informative for user